src/youtube-deno.ts

```diff
@@ -37,7 +33,7 @@ export class YouTube {
     this.contentHeaders["Content-Type"] = "application/json";
   }
 
-  private create_url(method: string, params?: schemas.param) {
```

Comment:
Instead of making it public, it'll be better to use `protected`, that way we'll safely be able to use it in the derived class.